src/base/seeds/View/Projects.svelte

```diff
@@ -49,7 +53,10 @@
 </style>
 
 <div class="projects">
-  <List items={projects} query={fetchMoreProjects}>
+  <List
+    bind:items={projects}
+    complete={projects.length === totalProjectCount}
```

Comment:
We should hide the button in case a project gets added while the user is paginating through the list.
```suggestion
    complete={projects.length >= totalProjectCount}
```